paasta_tools/kubernetes_tools.py

```diff
@@ -1369,14 +1369,20 @@ def list_deployments(
     tail_lines_dict: MutableMapping[str, List[str]] = {"stdout": [], "stderr": []}
     for container in pod.spec.containers:
         if container.name != HACHECK_POD_NAME:
-            tail_lines_dict["stdout"].append(
-                kube_client.core.read_namespaced_pod_log(
-                    name=pod.metadata.name,
-                    namespace=pod.metadata.namespace,
-                    container=container.name,
-                    tail_lines=num_tail_lines,
+            try:
+                tail_lines_dict["stdout"].append(
+                    kube_client.core.read_namespaced_pod_log(
+                        name=pod.metadata.name,
+                        namespace="paasta",
+                        container=container.name,
+                        tail_lines=num_tail_lines,
+                    )
                 )
-            )
+            except ApiException as e:
+                tail_lines_dict["stdout"].append(
+                    f"couldn't read stdout/stderr because {e.getResponseBody()}"
+                )
+
```

Comment:
This isn't really stdout.  Can we just follow the same pattern as we do for mesos and return a separate field for the error (https://github.com/Yelp/paasta/blob/v0.93.19/paasta_tools/mesos_tools.py#L456)?

That way the client can decide whether to, say, put the error in red or whatever.